src/TConsole.cpp

```diff
@@ -2164,7 +2164,7 @@ void TConsole::slot_adjustAccessibleNames()
         return;
     case Buffer:
         // This is not a visible thing so is not accessible to screen readers
-        [[fallthrough]];
+        break;
```

Comment:
Semantically the same, but gives a tiny bit clearer the intention.
```suggestion
        return;
```

I'll accept it as a quick fix to avoid the **intentional** termination that it was intended to give simply because we need to get the release out. However it was intended to give a firm "you can't do **that** with **this**" indication and I still think it would have been better for me to have coded it so Mudlet didn't try to do *that* in the first place...